core/templates/components/skill-selector/skill-selector.directive.html

```diff
@@ -62,12 +62,42 @@ <h3><label class="list-group-item-heading" style="padding-left:10px"><[topicName
   </div>
 </div>
 <style>
-  .skill-selector-parent .skill-list-box {
+  select-skill .skill-selector-parent
```

Comment:
Should there be a comma at the end of this line?

Reply:
Done.